setup.py

```diff
@@ -12,12 +12,16 @@
     print "You must have setuptools installed to use setup.py. Exiting..."
     raise SystemExit(1)
 
-test_requirements = [
+
+install_dependencies = (
```

Comment:
That's Important because requests are dependency of the main distribution of the api.